src/views/AddRecord.jsx

```diff
@@ -8,17 +8,84 @@ import { MaterialIcons } from "@expo/vector-icons";
 import MapView, { Marker } from "react-native-maps";
 import { AddRecordStyle } from "../themes/AddRecordStyle";
 import { mapDark } from "../themes/MapStyle";
+import * as ImagePicker from 'expo-image-picker';
+import { auth, storage, db} from '../api/firebase'
+import { doc, collection, query, getDocs, setDoc} from 'firebase/firestore'
+import { ref, uploadBytes } from 'firebase/storage'
+import 'react-native-get-random-values';
+import { v4 as uuidv4 } from 'uuid';
+import ButtonBlue from "../components/ButtonBlue";
+
+
 
 export class AddRecord extends Component {
-  state = {
-    markerLocation: null,
+  constructor(props) {
+    super(props);
+    this.state = {
+      image: null,
+      isLoading: false,
+      uploading: false,
+      descripcion: '',
+      markerLocation: {
+                latitude: 19.244,
+                longitude: -103.725
+      }
+    };
+  }
```

Comment:
Check the indentation of this file please

Reply:
done